### PR TITLE
structure and meta styling

### PR DESCRIPTION
**Pull request by chriswhong**
- Styles structure and meta tabs to match mockups
- Removes `.eslintcache` from version control

This adds `important: true` to `tailwind.css` which makes all tailwind styles `!important`.  This was necessary to override the default styles of `react-tooltip` for a tooltip used in the `LabeledStats` component.  If we don't want to go this route, we'll need to find another solution to make custom-styled tooltips.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -133,5 +133,6 @@ module.exports = {
   plugins: [
     require('@tailwindcss/forms'),
     require('@tailwindcss/line-clamp'),
-  ]
+  ],
+  important: true
```
Comment (b5) [inline tailwind.config.js]: I don't think this is a good idea. Can we instead style react-tooltip directly with normal CSS instead of using tailwind?

Comment (chriswhong) [reply tailwind.config.js]: Will do

Comment (chriswhong) [reply tailwind.config.js]: Neat, tailwind has an `@apply` that lets you just throw tailwind classes into your normal CSS.

```
.light-tooltip {
  @apply rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 opacity-100 p-4 !important;
}
```

File: src/features/dsComponents/schema/DynamicEditField.tsx [hunk 2]
```diff
@@ -140,7 +140,7 @@ const DynamicEditField: React.FunctionComponent<DynamicEditFieldProps> = ({
   }
 
   return (
-    <div style={{ minWidth }} className={classNames('overflow-hidden whitespace-pre min-h-full overflow-ellipsis', { 'invalid': !isValid, 'dynamic-edit-field-large': large, 'focused': focused, 'dynamic-edit-field-editable': editable })} >
+    <div style={{ minWidth }} className={classNames('whitespace-pre min-h-full', { 'invalid': !isValid, 'dynamic-edit-field-large': large, 'focused': focused, 'dynamic-edit-field-editable': editable })} >
```
Comment (b5) [inline src/features/dsComponents/schema/DynamicEditField.tsx]: would prefer to see this spread across multiple lines

File: src/features/dsComponents/meta/Meta.tsx [hunk 3]
```diff
@@ -28,9 +28,9 @@ const renderValue = (value: string | string[] | object) => {
   }
 }
 
-const renderChips = (value: string[] | undefined) => (
+export const renderChips = (value: string[] | undefined) => (
   <div>
-    {value && value.map((d, i) => (<span key={i} className='text-xs rounded bg-gray-300 px-2 py-1 mr-2'>{d}</span>))}
+    {value && value.map((d, i) => (<span key={i} className='leading-tight text-gray-400 text-xs tracking-wider inline-block border border-qrigray-400 rounded-md px-2 py-1 mr-1 mb-1'>{d}</span>))}
   </div>
 )
```
Comment (b5) [inline src/features/dsComponents/meta/Meta.tsx]: This should be made into a `Chips` or `MetaChips` component

File: src/features/dsComponents/structure/FormatConfigHistory.tsx [hunk 4]
```diff
@@ -13,11 +13,11 @@ const FormatConfigHistory: React.FunctionComponent<FormatConfigHistoryProps> = (
   const format = structure.format
   const formatConfig = structure.formatConfig
```
Comment (b5) [inline src/features/dsComponents/structure/FormatConfigHistory.tsx]: rename this component to just `FormatConfig`, add a `showTrueOptionsOnly` prop, & use it when calling from history. This component is super old 😄

File: src/features/dsComponents/schema/SchemaItem.tsx [hunk 5]
```diff
@@ -60,59 +60,75 @@ const SchemaItem: React.FunctionComponent<SchemaItemProps> = ({
...
+              placeholder='validation'
+              value={data.validation || ''}
+              onChange={handleDynamicEditChange}
+              allowEmpty expanded={expanded}
+              minWidth={100}
+              editable={editable}
+            />
+          </div>
+        </td>
+      */}
```
Comment (b5) [inline src/features/dsComponents/schema/SchemaItem.tsx]: let's remove this entirely

File: src/features/dsComponents/schema/SchemaItem.tsx [hunk 6]
```diff
@@ -60,59 +60,75 @@ const SchemaItem: React.FunctionComponent<SchemaItemProps> = ({
...
-          onChange={handleDynamicEditChange}
-          allowEmpty expanded={expanded}
-          minWidth={100}
-          editable={editable}
-        />
+      {/*
+        <td className={tdCellClassName} onClick={() => setExpanded((prev) => !prev)} >
+          <Icon icon={expanded ? 'caretDown' : 'caretRight'} size='md' color='medium'/>
+        </td>
+      */}
```
Comment (b5) [inline src/features/dsComponents/schema/SchemaItem.tsx]: remove this for now.

File: src/features/dsComponents/schema/Schema.tsx [hunk 7]
```diff
@@ -32,30 +33,53 @@ const Schema: React.FC<SchemaProps> = ({
...
+                description
+              </div>
+            </th>
+            {/*
+              <th className={thClassName}>
+                <div className={thCellClassName}>
+                  validation
+                </div>
+              </th>
+            */}
```
Comment (b5) [inline src/features/dsComponents/schema/Schema.tsx]: remove

File: src/chrome/LabeledStats.tsx [hunk 8]
```diff
@@ -30,40 +34,78 @@ export interface LabeledStatsProps {
...
 
+export function getStats (data: Structure): any[] {
+  return [
+    { 'label': 'format', 'value': data.format ? data.format.toUpperCase() : 'unknown' },
+    { 'label': 'body size', 'value': data.length ? fileSize(data.length) : '—' },
+    { 'label': 'entries', 'value': abbreviateNumber(data.entries) || '—' },
+    { 'label': 'errors', 'value': data.errCount ? abbreviateNumber(data.errCount) : '—' },
+    { 'label': 'depth', 'value': data.depth || '—' }
+  ]
+}
```
Comment (b5) [inline src/chrome/LabeledStats.tsx]: the stats prop is intended to be generic. Move this into `/src/qri/dataset.ts`, somewhere near the declaration of the structure component, restore the `Stat` interface, and pass the result of calling `getStats` to the component

Comment (chriswhong) [reply src/chrome/LabeledStats.tsx]: LabeledStats is rendering a tooltip that contains info from `structure`, so we need to pass structure in as a prop and call `getProps()` from within this component.  Either that or we need a separate component to render the tooltip that will select structure on its own, which feels wrong.

I've moved the `Stat` interface to `/qri/dataset.ts` and made sure that `getStats()` is returning `Stat[]`

**Pull request merged**